src/h3lib/lib/baseCells.c

```diff
@@ -305,7 +296,7 @@ const int baseCellNeighbor60CCWRots[NUM_BASE_CELLS][7] = {
  * This table can be accessed using the functions `_faceIjkToBaseCell` and
  * `_faceIjkToBaseCellCCWrot60`
  */
-static const BaseCellOrient faceIjkBaseCells[NUM_ICOSA_FACES][3][3][3] = {
+const BaseCellOrient faceIjkBaseCells[NUM_ICOSA_FACES][3][3][3] = {
```

Comment:
I allowed this to be imported in order to use it in the `generateBaseCellVertexRotations` script.

Reply:
Note this is something which will have trouble in Windows where that symbol won't get exported.

Reply:
I am only using it for the script, which I assume only gets used by developers (and hopefully doesn't need to get run again, including it only as reference). We use other `extern constant` tables in the same way.

Reply:
I can remove `extern` now, so this is moot